buildscripts/healthcheck.sh

```diff
@@ -22,6 +22,26 @@ _init () {
 }
 
 HealthCheckMain () {
+    # log the starting time
+    start=$(date +%s)
+    # initialize current time
+    current=$(date +%s)
+    # initialize diff
+    diff=$(( current - start ))
+    # we report healthy status for first 120s without sending curl requests. This just to make sure 
```

Comment:
Comments can follow a 80 column length rule so that they are consistently blocked together like here 

```
# We report healthy status for the first 120s without sending any curl requests. 
# This is just to make sure Minio server is running and can actually respond 
# to healthcheck curl requests. In distributed environment like Swarm, traffic 
# is routed to a container only when it reports a `healthy` status. 
#
# Refer here: https://github.com/moby/moby/pull/28938#issuecomment-301753272
```